### PR TITLE
Problem: There are unnecessary markers

### PR DESCRIPTION
**Pull request by codegeschrei**
Solution: remove the tendermint marker for tests

I deleted the test `test_get_blocks_by_txid_endpoint_returns_400_bad_query_params` since we already have it in `test_block`

### CODE REVIEW

File: tests/commands/test_commands.py [hunk 1]
```diff
@@ -42,22 +40,21 @@ def test_main_entrypoint(mock_start):
...
+@patch('bigchaindb.commands.bigchaindb._run_init')
+@patch('bigchaindb.config_utils.autoconfigure')
+def test_bigchain_run_start(mock_setup_logging, mock_run_init,
+                            mock_autoconfigure, mock_processes_start):
     from bigchaindb.commands.bigchaindb import run_start
     args = Namespace(config=None, yes=True,
                      skip_initialize_database=False)
     run_start(args)
-    mocked_setup_logging.assert_called_once_with(user_log_config=config['log'])
+    assert mock_setup_logging.called
```
Comment (z-bowen) [inline tests/commands/test_commands.py]: What's the motivation for the weaker assertion?

File: tests/commands/test_commands.py [hunk 2]
```diff
@@ -235,22 +223,19 @@ def test_run_start_when_db_already_exists(mocker,
...
+    mocked_start = mocker.patch('bigchaindb.start.start')
 
     def mock_run_init():
         raise DatabaseAlreadyExists()
     monkeypatch.setattr('builtins.input', lambda: '\x03')
     monkeypatch.setattr(
         'bigchaindb.commands.bigchaindb._run_init', mock_run_init)
     run_start(run_start_args)
-    mocked_setup_logging.assert_called_once_with(user_log_config=config['log'])
     assert mocked_start.called
```
Comment (z-bowen) [inline tests/commands/test_commands.py]: Same question here: why are we no longer asserting the `args`

File: tests/commands/test_commands.py (hunk 1 above)
Comment (codegeschrei) [reply tests/commands/test_commands.py]: The removed line does not work like this anymore. Before `setup_logging` had parameters you could provide but that's not the case anymore.

File: tests/commands/test_commands.py (hunk 2 above)
Comment (codegeschrei) [reply tests/commands/test_commands.py]: see above

File: tests/web/test_block_tendermint.py [hunk 3]
```diff
@@ -73,29 +71,3 @@ def test_get_blocks_by_txid_endpoint_returns_empty_list_not_found(client):
...
-    res = client.get(BLOCKS_ENDPOINT + '?transaction_id=123&foo=123')
-    assert res.status_code == 400
-    assert res.json == {
-        'message': 'Unknown arguments: foo'
-    }
-
-    res = client.get(BLOCKS_ENDPOINT + '?transaction_id=123&status=123')
-    assert res.status_code == 400
-    assert res.json == {
-        'message': 'Unknown arguments: status'
```
Comment (z-bowen) [inline tests/web/test_block_tendermint.py]: Why is the error response message different between these two tests? The one in `test_blocks` asserts 'Unknown arguments: status'. How did these ever both pass?

**Pull request merged**